pkg/internal/webhook/validator.go

```diff
@@ -115,9 +116,17 @@ func (v *validator) validate(ctx context.Context, obj runtime.Object) (admission
 			return nil, err
 		}
 		if !response.Allowed {
```

Comment:
```suggestion
		if !response.Allowed && len(response.Errors) == 0 {
		    el = append(el, errors.New("a plugin did not allow the CertificateRequest for unknown reasons"))
		}
		for _, e := range response.Errors {
		    el = append(el, e)
		}
```

Could you update the code as suggested, this would make it easier to understand for me.

Reply:
I think that we should not use field error here to avoid misleading users suggesting that there is something wrong with the spec if the error is about something different, i.e some internal error etc.

Reply:
Thanks for explaining, both comments make sense to me.